tools/pythonpkg/src/common/exceptions.cpp

```diff
@@ -34,88 +222,139 @@ void RegisterExceptions(const py::module &m) {
 	//       |__ NotSupportedError
```

Comment:
I can't comment on the actual line, but it looks like we've deviated from the spec in the above comment, which isn't good